### PR TITLE
Generate all the sample data from demo_data command

### PR DESCRIPTION
**Pull request by szok**
Change of the demo data generating to all fixtures

### CODE REVIEW

File: os/init.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ echo "GRANT ALL ON ralph.* TO ralph@'%' IDENTIFIED BY 'ralph'; FLUSH PRIVILEGES"
 /home/ralph/bin/ralph migrate --fake
 /home/ralph/bin/ralph createsuperuser ralph --noinput --user ralph --email ralph@allegrogroup.com
 /home/ralph/bin/ralph shell < /home/ralph/createsuperuser.py
-DJANGO_SETTINGS_MODULE=ralph.settings /home/ralph/bin/ralph make_demo_data -d assets_dc_assets -d relations
+DJANGO_SETTINGS_MODULE=ralph.settings /home/ralph/bin/ralph make_demo_data -d relations -d networks -d assets_dc_assets
```
Comment (mkurek) [inline os/init.sh]: maybe adding flag `--all` to make_demo_data will be better idea than adding here new demo every time there is sth new?

**Comment by xliiv** [comment]
Commit message doesn't explain why this change is made, beside that it's :ok:

**Comment by mkurek** [comment]
:ok:

**Comment by andrzej-jankowski** [comment]
:ok:

**Pull request merged**